setup.py

```diff
@@ -73,7 +73,7 @@ def is_rtd() -> bool:
     "codespell",
     "coverage",
     "isort",
-    "mypy",
+    "mypy==0.991",
```

Comment:
pinned this temporarily because of this error with mypy 1.0.0:

```
$ make test-mypy 
mypy setup.py craft_parts tests
setup.cfg:17: error: Error importing plugin "pydantic.mypy": could not convert string to float: '1.0.0'  [misc]
Found 1 error in 1 file (errors prevented further checking)
make: *** [Makefile:92: test-mypy] Error 2

Reply:
I think we need to pin all the linters on all our apps anyway and let Renovate handle updates